addons/account/wizard/account_move_send.py

```diff
@@ -386,7 +411,8 @@ def _prepare_invoice_pdf_report(self, invoice, invoice_data):
         if invoice.invoice_pdf_report_id:
             return
 
-        content, _report_format = self.env['ir.actions.report']._render('account.account_invoices', invoice.ids)
+        pdf_report_name = invoice_data['pdf_report_name']
+        content, _report_format = self.env['ir.actions.report']._render(pdf_report_name, invoice.ids)
```

Comment:
```suggestion
        pdf_report = self.env['ir.actions.report'].browse(invoice_data['pdf_report_id'])
        content, _report_format = self.env['ir.actions.report']._render(pdf_report.report_name, invoice.ids)
```
This is not perfect but we could improve the `_render` to allow passing directly the record